docs/DeveloperGuide.md

```diff
@@ -261,33 +261,31 @@ _{Explain here how the data archiving feature will be implemented}_
 **Target user profile**:
 Purchasing managers who are proficient with typing for small F&B businesses
 
-**Value proposition**:
+
+**Value proposition**:  
 This application will help small businesses to manage perishable goods within a single inventory
 (no support for multiple inventories).
 
-### User stories
 
+### User stories
 Add Inventory Item:
 
 Delete Inventory Item:
 
 Update inventory items:
-
 1. As a purchasing manager, I can rename my inventory items, so that I can update items with an incorrect name.
-1. As a purchasing manager, I can set the inventory quantity of my items.
-1. As a purchasing manager, I can set the date I bought my inventory, so that I know how long I have been keeping certain items.
-1. As a purchasing manager, I can set the date my inventory will expire, so that I know when certain items need to be consumed or used.
-1. As a purchasing manager, I can increase the inventory quantity of my items, so that I can keep my stock updated when I purchase new items.
-1. As a purchasing manager, I can decrease the inventory quantity of my items.
+2. As a purchasing manager, I can set the inventory quantity of my items.
+3. As a purchasing manager, I can set the date I bought my inventory, so that I know how long I have been keeping certain items.
+4. As a purchasing manager, I can set the date my inventory will expire, so that I know when certain items need to be consumed or used. 
+5. As a purchasing manager, I can increase the inventory quantity of my items, so that I can keep my stock updated when I purchase new items.
+6. As a purchasing manager, I can decrease the inventory quantity of my items.
```

Comment:
Let us follow the coding style and use generic numbers. Also, trailing whitespace was added for some reason, probably unintentional.

```suggestion
1. As a purchasing manager, I can set the inventory quantity of my items.
1. As a purchasing manager, I can set the date I bought my inventory, so that I know how long I have been keeping certain items.
1. As a purchasing manager, I can set the date my inventory will expire, so that I know when certain items need to be consumed or used.
1. As a purchasing manager, I can increase the inventory quantity of my items, so that I can keep my stock updated when I purchase new items.
6. As a purchasing manager, I can decrease the inventory quantity of my items.
```